filebeat/input/kafka/input_test.go

```diff
@@ -21,16 +21,13 @@ package kafka
 
 import (
 	"testing"
-
-	"github.com/elastic/beats/v7/filebeat/input/inputtest"
-	"github.com/elastic/beats/v7/libbeat/common"
 )
 
 func TestNewInputDone(t *testing.T) {
-	config := common.MapStr{
-		"hosts":    "localhost:9092",
-		"topics":   "messages",
-		"group_id": "filebeat",
-	}
-	inputtest.AssertNotStartedInputCanBeDone(t, NewInput, &config)
+	//config := common.MapStr{
```

Comment:
What's up with this?

Reply:
ah yes, wanted to discuss this. For the old style input there was the inputtest.AssertNotStartedInputCanBeDone(t, NewInput, &config) call I was wondering if under v2 input this test still makes sense and if there is an input.v2 equivalent (couldn't find it).